UnitTests/Onboarding/ContextualOnboarding/ContextualDaxDialogsFactoryTests.swift

```diff
@@ -59,11 +59,13 @@ final class ContextualDaxDialogsFactoryTests: XCTestCase {
     func testWhenMakeViewForSearchDoneWithShouldFollowUpThenOnboardingsearchDoneViewCreatedAndOnActionNothingOccurs() throws {
         // GIVEN
         var onDismissRun = false
+        var onGotItPressedRun = false
         let dialogType = ContextualDialogType.searchDone(shouldFollowUp: true)
         let onDismiss = { onDismissRun = true }
+        let onGotItPressed = {onGotItPressedRun = true }
```

Comment:
swiftlint will complain about this
```suggestion
        let onGotItPressed = { onGotItPressedRun = true }
```